pycparser/c_generator.py

```diff
@@ -138,11 +138,17 @@ def visit_Enum(self, n):
         return self._generate_struct_union_enum(n, name='enum')
 
     def visit_Enumerator(self, n):
-        return '{indent}{name} = {value},\n'.format(
-            indent=self._make_indent(),
-            name=n.name,
-            value=self.visit(n.value),
-        )
+        if not n.value:
```

Comment:
Is `n.value` an `int`, or a `str`?  If it's an `int`, `n.value` will be false if it's zero.

Reply:
Isn't `n.value` a node?

Reply:
That's right, its a node (pycparser.c_ast.Constant to be exact). So it will only be false if there is no node there, which is what we want.